src/services/juxt-web/routes/console/userpage.ts

```diff
@@ -289,13 +320,9 @@ async function userRelations(req, res, userID) {
 	} else {
 		followers = await database.getFollowedUsers(userContent);
 		communities = userContent.followed_communities;
-		communityMap = await util.getCommunityHash();
 		selection = 2;
 	}
 
-	if (followers[0] === '0') {
-		followers.splice(0, 0);
-	}
```

Comment:
`followers` can't contain `string` so this check would always fail

Reply:
I'll double check the database to make sure this is the case before accepting this. Since we didn't use TypeScript before, and because Mongo documents can be modified outside of tools like Mongoose, these types weren't *super* enforced. They *can* have data outside of the schema and it's types.

It seems like this was used as a way to clear the `followers` array for some reason...? I'm not really sure what the point of that was, I'll give Jemma a ping to ask